jmix-core/core/src/main/java/io/jmix/core/querycondition/PropertyCondition.java

```diff
@@ -291,5 +313,8 @@ public static class Operation {
         public static final String IN_LIST = "in_list";
         public static final String NOT_IN_LIST = "not_in_list";
         public static final String IN_INTERVAL = "in_interval";
+        public static final String IS_EMPTY = "is_empty";
+        public static final String MEMBER_OF_LIST = "member_of_list";
+        public static final String NOT_MEMBER_OF_LIST = "not_member_of_list";
```

Comment:
Please create an issue for Studio to support collection attributes for `<propertyFilter/>` component. Currently Studio doesn't suggest collection attributes and mark new operations as not supported for the given entity attribute.

![image](https://github.com/jmix-framework/jmix/assets/7888486/c3f6f85b-519e-49ed-ad8a-c299778ca4d3)

Reply:
https://youtrack.haulmont.com/issue/JST-4537/Support-x-to-many-property-conditions-of-filter-component